Formula/deno.rb

```diff
@@ -1,9 +1,9 @@
 class Deno < Formula
   desc "Command-line JavaScript / TypeScript engine"
   homepage "https://deno.land/"
-  url "https://github.com/denoland/deno.git",
-    :tag      => "v0.21.0",
-    :revision => "4e88ba9a114279b3969d5ccca1cca0f74c8fc1fd"
+  url "https://github.com/denoland/deno/releases/download/v0.22.0/deno_src.tar.gz"
+  version "0.22.0"
```

Comment:
```suggestion
```

Homebrew should detect the version from the URL automatically, please remove this.

Reply:
no it detects `src` as the version unfortunately

Reply:
ah, yes I see now. What's the difference between the source tarball release asset and the one github generates automatically? Is it possible to use that one? Or convince upstream to start adding version numbers in their release assets?

Reply:
The GitHub ones don't include files from submodules and are useless in this case therefore.

I could try to submit a PR to add version numbers in the releases (I'm the one who created the source tarball PR https://github.com/denoland/deno/pull/3203 too), but this is more complex as it sounds, because it would require a major refactor of their [install shell scripts](https://github.com/denoland/deno_install) to be compatible with these version numbers too.

Reply:
Ok, in that case I think this is good enough. I should PR homebrew/brew to fix version detection from a gh releases asset.